packages/visx-drag/src/Drag.tsx

```diff
@@ -120,6 +120,8 @@ export default class Drag extends React.Component<DragProps, DragState> {
             height={height}
             onMouseMove={this.handleDragMove}
             onMouseUp={this.handleDragEnd}
+            onTouchMove={this.handleDragMove}

```

Comment:
Since this change `Drag` was re-implemented/re-factored to use the `useDrag` hook so there are unfortunately conflicts now. I think an easy change might be to simply update `onMouseMove/onMouseUp` to `onPointerMove/onPointerUp`, similar to my suggestion in `@visx/brush`